packages/data-context/test/unit/sources/ProjectDataSource.spec.ts

```diff
@@ -114,3 +120,60 @@ describe('transformSpec', () => {
     expect(result).to.eql(actual)
   })
 })
+
+describe('findSpecs', () => {
+  const temporary = 'tmp'
+
+  const fixture = [
+    'component/App.spec.ts',
+    'component/App.cy.ts',
+    'component/App.cy.js',
+    'e2e/onboarding.spec.ts',
+    'e2e/onboarding.cy.ts',
+    'e2e/onboarding.cy.js',
+  ]
+
+  let ctx: DataContext
+
+  beforeEach(async () => {
```

Comment:
Cool, I wonder if we want a `createTestDataContext` method that does all this - I think a lot of tests might benefit from it.

Reply:
Yeah! we may want to move to a util, that can be really useful so we can create more unit tests for different dataSources and dataActions